ginga/AstroImage.py

```diff
@@ -142,7 +142,7 @@ def load_file(self, filepath, numhdu=None, naxispath=None):
             if '.' in name:
                 name = name[:name.rindex('.')]
             if numhdu is not None:
-                name += ('[%d]' % numhdu)
+                name += '[{0}]'.format(numhdu)
             self.set(name=name)
 
         self.set(path=filepath, idx=numhdu)
```

Comment:
> does this work for fitsio-based loader?

I tried both settings by modifying `general.cfg`. So, if testing the way I did actually grabs the setting from the config file, then the answer is yes.